docs/fastdds/dds_layer/subscriber/subscriber/subscriber.rst

```diff
@@ -3,3 +3,98 @@
 Subscriber
 ==========
 
+The :ref:`api_pim_subscriber_class` acts on behalf of one or several :ref:`dds_layer_subscriber_dataReader` objects
+that belong to it.
+It serves as a container that allows grouping different :ref:`dds_layer_subscriber_dataReader` objects under
+a common configuration given by the :ref:`dds_layer_subscriber_subscriberQos` of the :ref:`api_pim_subscriber_class`.
+
+:ref:`dds_layer_subscriber_dataReader` objects that belong to the same :ref:`api_pim_subscriber_class`
+do not have any other relation among each other beyond the :class:`SubscriberQos` of the :class:`Subscriber`
+and act independently otherwise.
+Specifically, a :ref:`api_pim_subscriber_class` can host :ref:`dds_layer_subscriber_dataReader` objects
+for different topics and data types.
+
+
+.. _dds_layer_subscriber_subscriberQos:
+
+SubscriberQos
+-------------
+
+:ref:`api_pim_subscriberqos` controls the behavior of the :ref:`dds_layer_subscriber_subscriber`.
+Internally it contains the following :class:`QosPolicy` objects:
+
+.. |presentation| replace:: :cpp:func:`presentation<eprosima::fastdds::dds::SubscriberQos::presentation>`
+.. |partition| replace:: :cpp:func:`partition<eprosima::fastdds::dds::SubscriberQos::partition>`
+.. |group_data| replace:: :cpp:func:`group_data<eprosima::fastdds::dds::SubscriberQos::group_data>`
+.. |entity_factory| replace:: :cpp:func:`entity_factory<eprosima::fastdds::dds::SubscriberQos::entity_factory>`
+
++----------------------------------------+----------------------+----------+
+| QosPolicy class                        | Accessor/Mutator     | Mutable  |
++========================================+======================+==========+
+| :ref:`api_pim_presentationqospolicy`   | |presentation|       | Yes      |
++----------------------------------------+----------------------+----------+
+| :ref:`api_pim_partitionqospolicy`      | |partition|          | Yes      |
++----------------------------------------+----------------------+----------+
+| :ref:`api_pim_groupdataqospolicy`      | |group_data|         | Yes      |
++----------------------------------------+----------------------+----------+
+| :ref:`api_pim_entityfactoryqospolicy`  | |entity_factory|     | Yes      |
```

Comment:
This is not writing the parenthesis on the name of the functions. I see that they are clickable links now. IDK if we want the parenthesis there